### ISSUE TITLE
Indentation regression in 0.43.0 when var/let type is specified

### ISSUE THREAD

**Issue by fphilipe** [nicklockwood/SwiftFormat#515]
I just installed version 0.43.0 and came across the following regression. The following code would **not** get changed in version 0.42.0:

```swift
let good = bar
    .method {
        a
    }

let bad: Type = bar
    .method {
        a
    }

var good = bar
    .method {
        a
    }

var bad: Type = bar
    .method {
        a
    }
```

On version 0.43.0, the `let/var` that specify the type get changed:

```swift
let good = bar
    .method {
        a
    }

let bad: Type = bar
    .method {
    a
}

var good = bar
    .method {
        a
    }

var bad: Type = bar
    .method {
    a
}
```

<details>
<summary>Diff</summary>

```diff
 let good = bar
     .method {
         a
     }
 
 let bad: Type = bar
     .method {
-        a
-    }
+    a
+}
 
 var good = bar
     .method {
         a
     }
 
 var bad: Type = bar
     .method {
-        a
-    }
+    a
+}
```

</details>

Both versions were tested with default settings.

**Comment by nicklockwood** [comment]
@fphilipe thanks for reporting, I'll get it fixed asap.

**Comment by nicklockwood** [comment]
@fphilipe fixed in 0.43.1
